package.json

```diff
@@ -81,6 +81,7 @@
     "history": "3.3.0",
     "hoist-non-react-statics": "2.5.4",
     "htm-to-json": "0.1.4",
+    "i18next-client": "1.11.4",
     "intl": "1.2.5",
```

Comment:
Could you comment in the PR how much both libs will add in size? Thx

Reply:
approx. 760Kb